src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. It's better to set styles only for elements that require them. If you need to set a global box-sizing, consider using `html { box-sizing: border-box; }` and then inherit from it using `*, *::before, *::after { box-sizing: inherit; }`.